scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -651,6 +638,52 @@ protected String prepareJobInput(Job job, String localInputFolder, String pushUR
         return push_url_updated;
     }
 
+    private String appendSlashIfNeeded(String url) {
+        if (url.endsWith("/")) {
+            return url;
+        }
+        return url + "/";
+    }
+
+    private void updateInputSpace(Job job, String inputFolder) throws NotConnectedException, PermissionException {
+        List<String> inputSpaces;
+        if (job.getInputSpace() != null) {
+            inputSpaces = Arrays.asList(Tools.dataSpaceConfigPropertyToUrls(job.getInputSpace()));
+        } else {
+            inputSpaces = getUserSpaceURIs();
+        }
+        StringBuilder inputSpace_url_updated = buildUpdatedSpaceConfiguration(inputFolder, inputSpaces);
+
+        job.setInputSpace(inputSpace_url_updated.toString());
+        log.debug("Input space of job " + job.getName() + " will be " + inputSpace_url_updated);
+    }
+
+    private StringBuilder buildUpdatedSpaceConfiguration(String folder, List<String> spaceUrls) {
+        StringBuilder inputSpace_url_updated = new StringBuilder();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)